components/detection/handDetection/src/handDetection.py

```diff
@@ -55,7 +55,8 @@
 #
 #
 
-import sys, traceback, IceStorm, subprocess, threading, time, Queue, os, copy
+import sys, traceback, IceStorm, subprocess, threading, time, os, copy
+from multiprocessing import Queue
```

Comment:
Please, chan you check that these are the same Queue. It's also possible that it's not used anymore. Can you check? thank you.